From: EnderDev <42723993+EnderDev@users.noreply.github.com>
Date: Tue, 22 Feb 2022 17:36:57 +0000
Subject: Fix font rendering on Linux

Mark every Linux distro as unknown so Thebes cannot categorise system fonts.

Signed-off-by: EnderDev <42723993+EnderDev@users.noreply.github.com>
---
 gfx/thebes/gfxFcPlatformFontList.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gfx/thebes/gfxFcPlatformFontList.cpp b/gfx/thebes/gfxFcPlatformFontList.cpp
index 483fbdfe38..e45162417d 100644
--- a/gfx/thebes/gfxFcPlatformFontList.cpp
+++ b/gfx/thebes/gfxFcPlatformFontList.cpp
@@ -1855,21 +1855,21 @@ gfxFcPlatformFontList::DistroID gfxFcPlatformFontList::GetDistroID() const {
             result = DistroID::Fedora;
           }
           break;
         }
       }
       fclose(fp);
     }
     return result;
   };
   static DistroID sResult = getDistroID();
-  return sResult;
+  return DistroID::Unknown;
 }
 
 FontVisibility gfxFcPlatformFontList::GetVisibilityForFamily(
     const nsACString& aName) const {
   switch (GetDistroID()) {
     case DistroID::Ubuntu:
       if (FamilyInList(aName, kBaseFonts_Ubuntu_20_04)) {
         return FontVisibility::Base;
       }
       if (FamilyInList(aName, kLangFonts_Ubuntu_20_04)) {
-- 
2.25.1

